src/interfaces.ts

```diff
@@ -0,0 +1,213 @@
+import { GraphQLErrorExtended } from './postgraphile/extendedFormatError'
+import { PluginHookFn } from './postgraphile/pluginHook'
+import { IncomingMessage, ServerResponse } from 'http'
+import { GraphQLSchema } from 'graphql'
+import { Pool } from 'pg'
+import jwt = require('jsonwebtoken')
+import { GraphQLError } from 'graphql/error'
+
+export namespace Postgraphile {
```

Comment:
types that were used in various places i moved into a name space.  for instance, `PostGraphileOptions` was almost fully duped in 2 places.  i defined it once, imported it back two places, and extended it as needed

Reply:
Capital g please 🤗

Reply:
a5b736f